src/levels.rs

```diff
@@ -390,6 +390,13 @@ impl Neg for Mv {
     }
 }
 
+impl PartialEq for Mv {
+    #[inline(always)]
+    fn eq(&self, other: &Self) -> bool {
+        self.as_bytes() == other.as_bytes()
```

Comment:
```suggestion
        // `#[derive(PartialEq)]` compares per-field with `&&`,
        // which isn't optimized well and isn't coalesced into wider loads.
        // Comparing all of the bytes at once optimizes better with wider loads.
        // See <https://github.com/rust-lang/rust/issues/140167>.
        self.as_bytes() == other.as_bytes()
```

How does this sound?  If it's good, could you put the same comment on each `impl`?